<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtToDoMainWindow</class>
 <widget class="QMainWindow" name="QtToDoMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2131</width>
    <height>1242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtToDoMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="personalPushButton">
         <property name="text">
          <string>个人中心</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="writePushButton">
         <property name="text">
          <string>写日记</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="introduce">
         <property name="text">
          <string>QtToDo: 
用于编写待办与日记的工具软件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="toDoWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="todoLabel">
            <property name="text">
             <string>代办列：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="currentRow">
             <number>-1</number>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="toDoEditWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>500</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="pageMenu">
    <property name="title">
     <string>页面</string>
    </property>
    <addaction name="actionhome"/>
    <addaction name="separator"/>
    <addaction name="actionpersonal"/>
    <addaction name="separator"/>
    <addaction name="actionwrite"/>
   </widget>
   <widget class="QMenu" name="otherMenu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="pageMenu"/>
   <addaction name="otherMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionhome">
   <property name="text">
    <string>主页</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionpersonal">
   <property name="text">
    <string>个人中心</string>
   </property>
  </action>
  <action name="actionwrite">
   <property name="text">
    <string>写日记</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
